android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -125,7 +127,10 @@ public void initKMKeyboard(final Context context) {
 
     // Normally, this would be true to prevent the WebView from accessing the network.
     // But this needs to false for sending embedded KMW crash reports to Sentry (keymanapp/keyman#3825)
-    getSettings().setBlockNetworkLoads(!KMManager.getMaySendCrashReport());
+    // Throws SecurityException if INTERNET permission not granted
```

Comment:
I'm not sure I understand this comment -- does this mean `hasInternetPermission` will crash?

Reply:
`setBlockNetworkLoads()` throws SecurityException if INTERNET permission is not granted. That's why I added a `KMManager.hasInternetPermission()` check.

Reply:
Okay, the comment should be moved inside the if() block then :grin: